src/lib/routes/admin-api/project/api-token.ts

```diff
@@ -81,8 +82,12 @@ export class ProjectApiTokenController extends Controller {
                 openApiService.validPath({
                     tags: ['Projects'],
                     operationId: 'getProjectApiTokens',
+                    summary: 'Get api tokens for project.',
+                    description:
+                        'Returns the [project API tokens](https://docs.getunleash.io/how-to/how-to-create-project-api-tokens) that have been created for this project.',
                     responses: {
                         200: createResponseSchema('apiTokensSchema'),
+                        ...getStandardResponses(401, 403),
```

Comment:
Wouldn't 404 also be a possibility if the project doesn't exist?

Reply:
Yeah I'm gonna have a look and see, it should be, but I'm not entirely sure. Some of the others here only return a 400

Reply:
This appears to just return an empty 200 which I find a bit odd

Reply:
Yeah fair enough. I can add a 404 standard response now, and set up a task for implementing it!